### ISSUE TITLE
Remote DB?

### ISSUE THREAD

**Issue by alex3t** [KiteSync/realm-sync-js#152]
Am I correctly understand that remote DB is DynamoDB instance? I did not find any references in docs

**Comment by alamuv** [comment]
Yes, that's right. We have used DynamoDB as the remote DB. I have added instructions on our wiki on how to setup Remote DB. https://github.com/KiteSync/realm-sync-js/wiki/3.-Backend 
I hope this helps you!

**Comment by alex3t** [comment]
Got it! Thanks
